clouddriver-event/src/main/kotlin/com/netflix/spinnaker/clouddriver/event/ComposedSpinnakerEvent.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.clouddriver.event
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+
+/**
+ * Marks a [SpinnakerEvent] as being constructed of multiple [SpinnakerEvent]s.
+ *
+ * This interface is necessary to correctly hydrate [EventMetadata] on [SpinnakerEvent] before persisting.
+ */
```

Comment:
Is this a 'Composite' or 'Composed' ?

Reply:
Will rename to `CompositeSpinnakerEvent`, much bester.